### PR TITLE
Init Layout

### PR DESCRIPTION
**Pull request by nampt-3121**
## Related Tickets
- [#51102](https://edu-redmine.sun-asterisk.vn/issues/51102)

## WHAT (optional)
- Init Layout

## HOW
-  Add i18n, bootstrap, jquery....
-  Build master layout

## WHY (optional)
- Init Layout

## Evidence (Screenshot or Video)
Screenshot from 2022-07-20 10-44-58

### CODE REVIEW

**Comment by nampt-3121** [comment]
ready

File: app/assets/stylesheets/color.scss [hunk 1]
```diff
@@ -0,0 +1,2 @@
+$white : #fff;
+$gray-medium-light: #eaeaea;
```
Comment (huongnt-2545) [inline app/assets/stylesheets/color.scss]: end file

File: app/assets/stylesheets/custom.scss [hunk 2]
```diff
@@ -0,0 +1,118 @@
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
+@import './color.scss';
```
Comment (huongnt-2545) [inline app/assets/stylesheets/custom.scss]:
```suggestion
@import 'color';
```

File: app/views/layouts/_rails_default.html.erb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<%= csrf_meta_tags %>
+<%= csp_meta_tag %>
+<meta charset="utf-8">
+
+<%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
+<%= javascript_pack_tag "application", "data-turbolinks-track": "reload" %>
```
Comment (huongnt-2545) [inline app/views/layouts/_rails_default.html.erb]: end file

File: app/views/layouts/_shim.html.erb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<!--[if lt IE 9]>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/r29/html5.min.js"></script>
+<![endif]-->
```
Comment (huongnt-2545) [inline app/views/layouts/_shim.html.erb]: end file

**Review by huongnt-2545 — COMMENTED** [review]
commented

**Comment by nampt-3121** [comment]
ready

File: app/helpers/application_helper.rb [hunk 5]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = t "base_title"
+    page_title.blank? ? base_title : page_title.concat(" | ").concat(base_title)
```
Comment (ngocvt-0484) [inline app/helpers/application_helper.rb]:
```suggestion
    page_title.blank? ? [page_title, base_title].join(" | ")
```

File: app/helpers/sessions_helper.rb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+module SessionsHelper
```
Comment (ngocvt-0484) [inline app/helpers/sessions_helper.rb]: remove

File: app/views/sessions/new.html.erb [hunk 7]
```diff
@@ -0,0 +1,13 @@
+<% provide :title, t(".title") %>
+<h1><%= t ".welcome" %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(:session, url: "login_path") do |f| %>
```
Comment (ngocvt-0484) [inline app/views/sessions/new.html.erb]:
```suggestion
    <%= form_for(:session, url: login_path) do |f| %>
```

**Comment by nampt-3121** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**